nilearn/datasets.py

```diff
@@ -26,6 +25,17 @@
 
 import nibabel
 
+# Define _get_time, which is used module-wide to get the current time.
+try:
+    # Use uptime (when available) to avoid reporting time while computer slept.
+    import uptime
+    if uptime.uptime() is None:
+        raise Exception("uptime is not supported by this OS.")
+    _get_time = uptime.uptime
```

Comment:
I am just wondering if we should bother about uptime. My point is that yes it makes the total download time more accurate but:
* it does so only in a very restricted edge case: you have to interrupt the download of a dataset and shut down your computer for a while
* it's not like the dataset download time is a critical piece of information and I must say I have probably never paid attention to it ...
* uptime is quite an obscure 3rd party package and it is very unlikely to be installed on anyone's machine.